Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -128,7 +134,7 @@ - (void)beginLocationUpdates
 {
   if (!_locationManager) {
     _locationManager = [CLLocationManager new];
-    _locationManager.distanceFilter = RCT_DEFAULT_LOCATION_ACCURACY;
+    _locationManager.distanceFilter = _observerOptions.distanceFilter;
```

Comment:
@christopherdro should we move `_locationManager.distanceFilter = _observerOptions.distanceFilter ` outside of the `if (!_locationManager)` so this could be changed when stoping and watching the location again? Also it seems that `beginLocationUpdatesWithDesiredAccuracy` is called from `getCurrentPosition` and it is not initialized from `beginLocationUpdatesWithDesiredAccuracy` would this be an issue if the value is nil or just be a noop?